packages/components/inputs/search-select-input/src/search-select-input.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { SafeHTMLElement } from '@commercetools-uikit/utils';
+import AsyncSelectInput from '@commercetools-uikit/async-select-input';
+import {
+  CustomSelectInputOption,
+  SearchIconDropdownIndicator,
+} from '@commercetools-uikit/select-utils';
+import messages from './messages';
+import { SearchSelectInputWrapper } from './search-select-input.styles';
+
+const SearchSelectInput = (props) => {
+  const intl = useIntl();
+  const noOptionsMessage =
+    props.noOptionsMessage ||
+    (() => intl.formatMessage(messages.noOptionsMessage));
+  const loadingMessage =
+    props.loadingMessage || intl.formatMessage(messages.loadingOptionsMessage);
+  const placeholder =
+    props.placeholder || intl.formatMessage(messages.placeholderMessage);
+
+  // eslint-disable-next-line react/display-name
+  const OptionComponent = (optionInnerProps) => (
```

Comment:
We still re-define this component at each render within `SearchSelectInput` which will mean it will mount-and unmount a lot to my understanding. Can't be just move it to L12?